gensim/test/test_similarity_metrics.py

```diff
@@ -240,36 +240,6 @@ def test_distributions(self):
         self.assertAlmostEqual(expected, result)
 
 
-class TestSoftCosineSimilarity(unittest.TestCase):
```

Comment:
How is `softcossim`'s replacement, `inner_product` itself, tested? Did we use to double-test it?

Reply:
There are 63 unit tests in [`test_similarities.py`](https://github.com/RaRe-Technologies/gensim/blob/develop/gensim/test/test_similarities.py#L1039).